add-chain/testdata/superchain/extra/genesis-system-configs/sepolia/expected_zorasep.json

```diff
@@ -0,0 +1,8 @@
+{
+  "batcherAddr": "0x3Cd868E221A3be64B161D596A7482257a99D857f",
+  "overhead": "0x00000000000000000000000000000000000000000000000000000000000000bc",
+  "scalar": "0x00000000000000000000000000000000000000000000000000000000000a6fe0",
+  "gasLimit": 30000000,
+  "baseFeeScalar": 0,
+  "blobBaseFeeScalar": 0
```

Comment:
I manually added these fields because they were non-existent previously.

Reply:
I think they didn't activate ecotone at genesis, so we wouldn't expect these fields to be present?

Reply:
Those fields are required by the `superchain` package due to the way the struct is defined [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/superchain.go#L56-L57). Should we make them optional by changing the json tags for those fields so that they include `omitempty`?

Reply:
Hmm I think perhaps we want them to be pointers, so that the default value is `nil`. Otherwise a missing field is indistinguishable from a field specifying `0`.

Reply:
Needed to make those fields points and add the `omitempty` tag to get this to work: https://github.com/ethereum-optimism/superchain-registry/pull/354/commits/ce93f6f20cc5405c613827fbeba563925f2e02da